### ISSUE TITLE
Interfaces are not exposed

### ISSUE THREAD

**Issue by hkamran80** [auth0/auth0-vue#76]
### Describe the problem

When I try to build my Vue 3 demo project with this library, I get the following errors.

```
node_modules/@auth0/auth0-vue/dist/typings/client.proxy.d.ts:2:32 - error TS2307: Cannot find module './interfaces' or its corresponding type declarations.

2 import { Auth0VueClient } from './interfaces';
                                 ~~~~~~~~~~~~~~

node_modules/@auth0/auth0-vue/dist/typings/global.d.ts:3:15 - error TS2307: Cannot find module './interfaces' or its corresponding type declarations.

3 export * from './interfaces';
                ~~~~~~~~~~~~~~

node_modules/@auth0/auth0-vue/dist/typings/index.d.ts:2:10 - error TS2305: Module '"./global"' has no exported member 'Auth0VueClient'.

2 import { Auth0VueClient, Auth0Plugin, Auth0PluginOptions, Auth0VueClientOptions } from './global';
           ~~~~~~~~~~~~~~

node_modules/@auth0/auth0-vue/dist/typings/index.d.ts:2:39 - error TS2305: Module '"./global"' has no exported member 'Auth0PluginOptions'.

2 import { Auth0VueClient, Auth0Plugin, Auth0PluginOptions, Auth0VueClientOptions } from './global';
                                        ~~~~~~~~~~~~~~~~~~

node_modules/@auth0/auth0-vue/dist/typings/index.d.ts:2:59 - error TS2305: Module '"./global"' has no exported member 'Auth0VueClientOptions'.

2 import { Auth0VueClient, Auth0Plugin, Auth0PluginOptions, Auth0VueClientOptions } from './global';
                                                            ~~~~~~~~~~~~~~~~~~~~~

node_modules/@auth0/auth0-vue/dist/typings/plugin.d.ts:2:59 - error TS2307: Cannot find module './interfaces' or its corresponding type declarations.

2 import { Auth0PluginOptions, Auth0VueClientOptions } from './interfaces';
                                                            ~~~~~~~~~~~~~~

node_modules/@auth0/auth0-vue/dist/typings/token.d.ts:2:32 - error TS2307: Cannot find module './interfaces' or its corresponding type declarations.

2 import { Auth0VueClient } from './interfaces';
                                 ~~~~~~~~~~~~~~
```

In the NPM package, the missing files are in `src/interfaces`, but they don't exist in the `dist/typings` directory. 

<img width="296" alt="Screen Shot 2022-02-24 at 17 25 24" src="https://user-images.githubusercontent.com/16967687/155635886-3c42773d-39a9-4e47-8d1c-caa29a7b4928.png">

### What was the expected behavior?

I expected no typing errors.

### Reproduction
1. Create a Vue 3 project with TypeScript (I used the Vite preset) and install this library
2. Import the library
3. Try to build the project

**Can the behavior be reproduced using the [Vue SDK Playground](https://github.com/auth0/auth0-vue/blob/master/DEVELOPMENT.md#the-sdk-playground)?**

No, because the playground uses the local files from the repository (where it is able to import the interfaces), whereas the NPM package uses a compiled version that does not have them.

### Environment

- **Version of `auth0-vue` used:** 1.0.0-beta.0
- **Version of `vue` used:** 3.2.25
- **Which browsers have you tested in?** N/A
- **Other modules/plugins/libraries that might be involved:** N/A

**Comment by frederikprijck** [comment]
Thanks, looking into this!

**Comment by frederikprijck** [comment]
I have created a PR that should solve this. Would u be able to give it a try and see if it solves this for u?

You can try this by doing:

- git clone the repository
- `npm i
- `npm pack`

Then in your project:
- `npm uninstall @auth0/auth0-vue` in your project
- `npm install ../../path/to/auth0-auth0-vue-1.0.0-beta.0.tgz`

And see if it builds.

**Comment by hkamran80** [comment]
It worked perfectly! Thank you!

**Comment by frederikprijck** [comment]
It's released as `1.0.0-beta.1`. 

Thanks again for reporting.
